packages/create/src/Generator.js

```diff
@@ -1,11 +1,23 @@
 import path from 'path';
+import commandLineArgs from 'command-line-args';
+
 import { copyTemplates, copyTemplate, copyTemplateJsonInto, installNpm } from './core.js';
+import { parseCliOptions } from './helpers.js';
+
+const optionDefinitions = [
+  { name: 'tag-name', type: String, defaultValue: '' },
+  { name: 'no-npm', type: Boolean, defaultValue: false },
```

Comment:
I would like to (later also) add a yes/no question for npm installation asa well... so let's use already use
```
{ name: 'install-npm', type: String, defaultValue: '' },
```

Reply:
oki

Reply:
fixed, we just have to uncomment line 44 in `Generator.js` and set the default to `{ name: 'npm-install', type: String, defaultValue: '' },` later.

Reply:
sound good 👍